DSharpPlus.CommandsNext/CommandsNextExtension.cs

```diff
@@ -236,8 +236,7 @@ private async Task HandleCommandsAsync(DiscordClient sender, MessageCreateEventA
 
             _ = Task.Run(async () => await this.ExecuteCommandAsync(ctx).ConfigureAwait(false));
 
-            if (this.Config.SignalHandledOnCommandFound)
-                e.Handled = true;
+            e.Handled = this.Config.MarkEventAsHandled;
```

Comment:
You missed my followup comment about this.

Reply:
If I make it negated, it will then by default work as "true", which is not backwards compatible.

Reply:
> **whether remaining MessageCreate event handlers __will run__**

In it's current state setting it to true [in the configuration] will actually prevent event handlers from running.